src/main/java/net/dries007/tfc/world/classic/worldgen/vein/VeinRegistry.java

```diff
@@ -107,31 +106,37 @@ public void postInit()
         weightedVeinTypes.clear();
         veinTypeRegistry.clear();
 
-        Path[] recursivePathList;
+        // Changed to only read files in the same folder, not subdirectories
+        File[] allFiles;
         try
         {
-            recursivePathList = Files.walk(tfcConfigDir.toPath()).filter(Files::isRegularFile).toArray(Path[]::new);
+            allFiles = tfcConfigDir.listFiles();
+            Objects.requireNonNull(allFiles);
         }
-        catch (IOException e)
+        catch (NullPointerException e)
         {
-            TerraFirmaCraft.getLog().error("Unable to read files in the config directory! TFC will not generate any ore veins!");
+            TerraFirmaCraft.getLog().error("Unable to read files in the damage resistance directory! TFC will not set entities resistance data!");
```

Comment:
copy-paste-typo

Reply:
That error output is actually at the right location (`VeinRegistry`). The one that is in `DamageResistanceRegistry` is set correctly too